packages/api/src/routers/api/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
this is to enable strict null checks in `routes/api` to ensure zod type inference works as expected (otherwise it sets every field as `| undefined`). we prob should enable it for the rest of the codebase

Reply:
ugh ci wasn't happy about multiple tsconfig. had to enable strict null checks globally